src/en/RainOfSnow.lua

```diff
@@ -15,10 +32,10 @@ local function expandURL(url)
 end
 
 local function getPassage(chapterURL)
-	local chap = GETDocument(chapterURL):selectFirst("div.content")
-	-- Removes the title and makes it an H1 for consistant custom CSS.
-	local title = chap:selectFirst("h2"):text()
-	chap:selectFirst("h2"):remove()
+	-- The additional shrinkURL is to support older version of the extension. (That didn't have shortened links)
+	local content = GETDocument(expandURL(shrinkURL(chapterURL)))
```

Comment:
expandURL on a shrinkURL? Is this a workaround to make already in library novels still work? If so, people will have to true delete the chapters anyway so might as well make them re-add them imo.

Reply:
Removed that work around.